Client/Frontend/Home/HomepageContextMenuHelper.swift

```diff
@@ -78,9 +78,14 @@ class HomepageContextMenuHelper: HomepageContextMenuProtocol {
 
     // MARK: - Default actions
 
-    func getOpenInNewPrivateTabAction(siteURL: URL) -> PhotonRowActions {
+    func getOpenInNewPrivateTabAction(siteURL: URL, sectionType: HomepageSectionType) -> PhotonRowActions {
         return SingleActionViewModel(title: .OpenInNewPrivateTabContextMenuTitle, iconString: ImageIdentifiers.newPrivateTab) { _ in
             self.delegate?.homePanelDidRequestToOpenInNewTab(siteURL, isPrivate: true)
+            if sectionType == .topSites {
+                TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .newPrivateTab, value: .topSite)
+            } else {
+                TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .newPrivateTab, value: .pocketSite)
+            }
```

Comment:
This would send telemetry for pocket if we have another action added in context menu helper 🤔 It would feel safer if this wasn't in an `else` case. I wonder if instead we could have something like (or better?):
```
if let telemetryValue = sectionType.newPrivateTabActionTelemetry {
    /// send telemetry
}
```